### PR TITLE
fix: upgrade to wasm-bindgen 0.2.89

### PR DESCRIPTION
**Pull request by iuioiua**
`wasm-bindgen@0.2.88` was yanked right after this module updated to it yesterday 😆

I don't want to give the impression I'll be making these updates for every little change. This one seems appropriate, as the current version was yanked.

### CODE REVIEW

File: lib/wasmbuild.generated.js [hunk 1]
```diff
@@ -222,8 +222,8 @@ const imports = {
   },
 };
 
-import { Loader } from "../loader.ts";
-import { cacheToLocalDir } from "../cache.ts";
+import { Loader } from "file:///Users/asher/GitHub/wasmbuild/loader.ts";
+import { cacheToLocalDir } from "file:///Users/asher/GitHub/wasmbuild/cache.ts";
```
[the remark this answers is not in the document]
Comment (iuioiua) [reply lib/wasmbuild.generated.js]: Yeah, I'm thinking the same. Let me look into that.

**Comment by iuioiua** [comment]
I was going to upgrade std too. Shall I do that in another PR?

**Review by dsherret — APPROVED** [review]
Thanks!

**Comment by iuioiua** [comment]
I have a couple of more updates, so maybe best to hold off before publishing a new version.

**Comment by dsherret** [comment]
Sure, sounds good and sorry!

**Pull request merged**